pytorch_toolkit/light-weight-face-anti-spoofing/models/mobilenetv3.py

```diff
@@ -0,0 +1,310 @@
+                                                                type_dropout=self.type_dropout))
+            input_channel = output_channel
+        self.features = nn.Sequential(*layers)
+        # building last several layers
+        self.conv_last = conv_1x1_bn(input_channel, embeding_dim)
+        self.avgpool = nn.AdaptiveAvgPool2d((1, 1))
+        # bulding heads for multi task
+        self.spoofer = nn.Sequential(
+            Dropout(p=prob_dropout_linear,
+                    mu=self.mu,
+                    sigma=self.sigma,
+                    dist=self.type_dropout,
+                    linear=True),
+            nn.BatchNorm1d(embeding_dim),
+            h_swish(),
+            nn.Linear(embeding_dim, 2),
+        )
+        if self.multi_heads:
+            self.lightning = nn.Sequential(
+                Dropout(p=prob_dropout_linear,
+                        mu=self.mu,
+                        sigma=self.sigma,
+                        dist=self.type_dropout,
+                        linear=True),
+                nn.BatchNorm1d(embeding_dim),
+                h_swish(),
+                nn.Linear(embeding_dim, 5),
+            )
+            self.spoof_type = nn.Sequential(
+                Dropout(p=prob_dropout_linear,
+                        mu=self.mu,
+                        sigma=self.sigma,
+                        dist=self.type_dropout,
+                        linear=True),
+                nn.BatchNorm1d(embeding_dim),
+                h_swish(),
+                nn.Linear(embeding_dim, 11),
+            )
+            self.real_atr = nn.Sequential(
+                Dropout(p=prob_dropout_linear,
+                        mu=self.mu,
+                        sigma=self.sigma,
+                        dist=self.type_dropout,
+                        linear=True),
+                nn.BatchNorm1d(embeding_dim),
+                h_swish(),
+                nn.Linear(embeding_dim, 40),
+            )
+
+    def forward(self, x):
+        x = self.features(x)
+        x = self.conv_last(x)
+        return x
+
+    def forward_to_onnx(self,x):
+        x = self.features(x)
+        x = self.conv_last(x)
+        x = self.avgpool(x)
+        x = x.view(x.size(0), -1)
+        spoof_out = self.spoofer(x)
+        if isinstance(spoof_out, tuple):
+            spoof_out = spoof_out[0]
+        probab = F.softmax(spoof_out*self.scaling, dim=-1)
+        return probab
+
+    def make_logits(self, features):
+        output = self.avgpool(features)
+        output = output.view(output.size(0), -1)
+        spoof_out = self.spoofer(output)
+        if self.multi_heads:
+            type_spoof = self.spoof_type(output)
+            lightning_type = self.lightning(output)
+            real_atr = torch.sigmoid(self.real_atr(output))
+            return spoof_out, type_spoof, lightning_type, real_atr
+        return spoof_out
+
+    def spoof_task(self, features):
```

Comment:
Methods `make_logits`, `spoof_task`, etc looks similar... Is it better to move them into some parent class?